react/README.md

```diff
@@ -89,6 +89,11 @@
     }
  
     export default ReservationCard;
+
+
+    // better
+    export default class ReservationCard extends React.Component {
```

Comment:
this is totally great! The only time we don't `export` a `class` inline, default or otherwise, is when we have static properties added prior to export (pending the advancement of the static class properties proposal)